mne/io/open.py

```diff
@@ -255,7 +257,6 @@ def _show_tree(fid, tree, indent, level, read_limit, max_str):
                 out += [next_idt + prepend + str(k) + ' = ' +
                         '/'.join(this_type) + ' (' + str(size) + ')' +
                         postpend]
-                out[-1] = out[-1].replace('\n', u'¶')
```

Comment:
You can use just '¶' (or maybe u'\xc2\xb6'.encode('utf-8') to be clear as the sky) instead of u'¶'.
If you use u'¶' it will try to decode out[-1] to unicode with ascii codec to make replace possible thus giving DecodeError.